storage/ctab/src/main/java/org/openscience/cdk/io/MDLV3000Reader.java

```diff
@@ -267,18 +298,29 @@ private void finalizeMol(ReadState state) {
                 }
             }
             if (valence < 0) {
-                isQuery = true;
-                logger.warn("Cannot set valence for atom with query bonds"); // also counts aromatic bond as query
+                // update variable so that it now considers both (1) presence of a query feature in atom container
+                // and (2) presence of an aromatic bond
+                isQueryOrAromaticBond = true;
+                logger.warn("Cannot set valence for atom with query bonds (this includes aromatic bonds)");
             } else {
-                final int unpaired = readData.getConnectedSingleElectronsCount(atom);
+                final int unpaired = readAtomContainer.getConnectedSingleElectronsCount(atom);
                 applyMDLValenceModel(atom, valence + unpaired, unpaired);
             }
         }
 
-        if (!isQuery)
-            finalizeStereochemistry(state, readData);
+        // skip infering and setting stereo chemistry if atom container has query features or aromatic bond
+        if (!isQueryOrAromaticBond)
```

Comment:
Not for this PR but we should tweak the stereo at some point, I think the reason it is skipped is the stereo assignment looks for 4 single bonds for tetrahedral (for example)